daprdocs/content/en/dotnet-sdk-docs/dotnet-actor.md

```diff
@@ -0,0 +1,31 @@
+---
+type: docs
+title: "Getting started with the Dapr actor .NET SDK"
+linkTitle: "Actor"
+weight: 20000
+description: How to get up and running with the Dapr .NET SDK
+---
+
+The Dapr actor package allows you to interact with Dapr virtual actors from a Python application.
+
+## Prerequisites
+
+- [Dapr CLI]({{< ref install-dapr-cli.md >}}) installed
+- Initialized [Dapr environment]({{< ref install-dapr-selfhost.md >}})
+- [.NET 5.0+](https://dotnet.microsoft.com/download) installed
```

Comment:
```suggestion
- [.NET 3.1+](https://dotnet.microsoft.com/download) installed
```

.NET 5.X is required to build **this** repo. 3.1 or newer is required for building applications with Dapr.